torch/nn/modules/loss.py

```diff
@@ -13,9 +13,10 @@ def _assert_no_grad(variable):
 
 
 class _Loss(Module):
-    def __init__(self, size_average=True):
+    def __init__(self, size_average=True, reduce=True):
```

Comment:
This is going to make all loss functions accept the `reduce` argument, while only `MSELoss` really supports it. Either only add it to `MSELoss`, or patch all other functions to raise an error if it's set to `False`